src/run.py

```diff
@@ -47,9 +47,7 @@ def echo(message):
             )
 
     def send_message(self, chat_id, text, reply_markup=None, emojize=True):
-        """
-        Send message to telegram bot.
-        """
```

Comment:
If you're deleting a wrong documentation, you should replace it with the correct one.

Reply:
is it  ok to write  '"" send message to telegram user""" ?

Reply:
I think yes. It's okay to replace it with correct docstring.